### DISCUSSION TITLE
I don't see the title

### DISCUSSION THREAD

**Discussion by Bowriverstudio** [wpengine/atlas-content-modeler#319] [Q&A / Help / Support] [answered]
I created a model - called Services.  When I create a few services I do not see the title.  How do I resolve this?

<img width="962" alt="Screen Shot 2021-10-08 at 2 51 14 PM" src="https://user-images.githubusercontent.com/36542929/136624460-18d424f9-65a3-4827-b45d-4a5f12b5f601.png">

**Comment by nickcernis** [comment, marked as answer]
Thanks for your question, @Bowriverstudio.

To fix this:

1. Make sure your Services model has at least one Text field.
2. When you create or edit the Text field, tick “use this field as the entry title”:
    <img width="762" alt="Screenshot 2021-10-11 at 13 59 46" src="https://user-images.githubusercontent.com/647669/136786576-4dcd0a99-fa94-4826-8884-d11939f56c78.png">
3. Edit existing services such as “No Title 181” and make sure the Text field you created has a value.

You should then see the service titles on the services listing page, instead of “No Title…”.

You can find more information about this and other field options in the [Atlas Content Modeler developer docs](https://developers.wpengine.com/docs/atlas-content-modeler/tutorial/models/fields/text/introduction).
